docs/language/ql-training/java/program-representation-java.rst

```diff
@@ -2,11 +2,7 @@
 Program representation
```

Comment:
[Line 26](https://github.com/Semmle/ql/pull/2257/files#diff-e4161e72680fe0e0cf7d21153fd0a09fR26) of this file also mentions "QL class".

Reply:
Similarly, replace QL with CodeQL on [line 50](https://github.com/Semmle/ql/pull/2257/files#diff-e4161e72680fe0e0cf7d21153fd0a09fR50).

Reply:
... and remove QL (or replace with CodeQL) on lines [77](https://github.com/Semmle/ql/pull/2257/files#diff-e4161e72680fe0e0cf7d21153fd0a09fR77) and [79](https://github.com/Semmle/ql/pull/2257/files#diff-e4161e72680fe0e0cf7d21153fd0a09fR79).  (I _think_ that's all for this file!)

Reply:
@lcartey: line 50 says: _QL also classifies `void` and `<nulltype>` (the type of the `null` literal) as primitive types._

Are these primitive types specific to Java analysis?